resources/meson.build

```diff
@@ -111,6 +111,7 @@ if host_machine.system() == 'nx'
                 '--exclude', '**/*.spv',
                 '--exclude', '**/meson.build',
             ],
+            input : essl_targets,
```

Comment:
I don't think this is the right way to do this; `input:` is for inputs used on the commandline via the `@INPUT@` placeholder(s). You want `depends:` instead.

Reply:
Replaced `input `with `depends`, the archive built is not empty as expected (even on a clean build)